scripts/utils.js

```diff
@@ -25,11 +25,21 @@ function wmAvailabilityCheck(url, onsuccess, onfail) {
  * @param url {string}
  * @return {bool}
  */
-function isValidSnapshotUrl(url) {
+function isValidUrl(url) {
   return ((typeof url) === "string" &&
     (url.indexOf("http://") === 0 || url.indexOf("https://") === 0));
 }
 
+// Function to check whether it is a valid URL or not
+function isNotExcludedUrl(url, blacklist) {
```

Comment:
Why provide the `blacklist` as a parameter each time we call `isNotExcluded`.
AFAIK we only have one blacklist and its not going to change.
Let's define blacklist just over `isNotExcludedUrl` method and then use it in our method.
So, the method will be called like `isNotExcludedUrl(url)`.
Is this OK?

Reply:
The only issue I see with this is that if we want to (down the line) implement a way to have the user add to the blacklist, would there be any issues with the blacklist being in utils.js instead of background.js?

Reply:
That is another feature, independent from this one.

We should not mix the two. These URLs we are trying to exclude now should be excluded in any case.

We'll make another method for the feature you describe if necessary.